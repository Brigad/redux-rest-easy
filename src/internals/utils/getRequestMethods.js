import { getNetworkHelpers } from '../network/networkHelpers';

const getRequestMethods = () => {
  const {
    requestDELETE,
    requestGET,
    requestPATCH,
    requestPOST,
    requestPUT,
  } = getNetworkHelpers();

  return {
    GET: requestGET,
    PATCH: requestPATCH,
    PUT: requestPUT,
    POST: requestPOST,
    DELETE: requestDELETE,
  };
};

export default getRequestMethods;
