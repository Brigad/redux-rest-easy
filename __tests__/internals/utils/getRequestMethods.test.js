import getRequestMethods from '../../../src/internals/utils/getRequestMethods';

describe('getRequestMethods', () => {
  test('only path', () => {
    expect(getRequestMethods()).toMatchSnapshot();
  });
});
