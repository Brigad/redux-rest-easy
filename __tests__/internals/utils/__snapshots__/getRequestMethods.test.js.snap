// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`getRequestMethods only path 1`] = `
Object {
  "DELETE": [Function],
  "GET": [Function],
  "PATCH": [Function],
  "POST": [Function],
  "PUT": [Function],
}
`;
